packages/vite/src/node/utils.ts

```diff
@@ -18,6 +18,13 @@ export function slash(p: string): string {
   return p.replace(/\\/g, '/')
 }
 
+export async function usePromiseAll<T, K>(
```

Comment:
**suggestion (dx):** Maybe we can use better generics here like `Item` and `Result`